micro-rdk/src/esp32/camera.rs

```diff
@@ -51,32 +85,28 @@ impl Esp32Camera {
             last_grab: t.now(),
         }
     }
-    pub fn setup(&self) -> Result<(), CameraError> {
-        let ret = (unsafe { crate::esp32::esp_idf_svc::sys::esp_camera_init(&self.config) })
-            as crate::esp32::esp_idf_svc::sys::esp_err_t;
+    fn setup(&self) -> Result<(), CameraError> {
```

Comment:
The `from_config` function above seems to be the only caller, and this `fn` isn't pub. Maybe just pull the logic into `from_config`?

Reply:
Yup! that's next step when I make the pin settings come dynamic